storehaus-hbase/src/main/scala/com/twitter/storehaus/asynchbase/AsyncHBaseLongStore.scala

```diff
@@ -63,6 +64,6 @@ class AsyncHBaseLongStore(protected val quorumNames: Seq[String],
   /** Close this store and release any resources.
     * It is undefined what happens on get/multiGet after close
     */
-  override def close(t: Time) = futurePool { client.shutdown() }
+  override def close(t: Time): Future[Unit] = futurePool { client.shutdown() }
```

Comment:
same comment here. In fact, I think this is more wrong since I think this is not actually waiting till the client has closed.